Modules/Registration/Common/test/itkMattesMutualInformationImageToImageMetricTest.cxx

```diff
@@ -214,6 +214,31 @@ int TestMattesMetricWithAffineTransform(
   metric->SetUseCachingOfBSplineWeights( useCachingBSplineWeights );
   metric->ReinitializeSeed(121212);
 
+  metric->SetFixedImageSamplesIntensityThreshold(100);
```

Comment:
I have a feeling these new tests could be written more compactly using testing macros:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/TestKernel/include/itkTestingMacros.h